templates/overrides/navigation/oe-corporate-blocks-eu-footer.html.twig

```diff
@@ -1,110 +1,97 @@
-{#
-/**
- * @file
- * Default theme implementation for rendering corporate blocks footer.
- *
- * Available variables:
- * - corporate_footer: Array with structure compatible to ECL ec-components-footer component.
- *   - contact: EU contact data for corporate footer.
- *   - social_media_title: Title for social media links column.
- *   - social_media: Social media links for corporate footer.
- *   - institution_links_title: Title for institution links column.
- *   - institution_links: EU institutions links for corporate footer.
- *   - legal_links_title: Title for legal links column.
- *   - legal_links: Legal links for corporate footer.
- *
- * @ingroup themeable
- */
-#}
-
-{% set _section_2 = [] %}
-{% set _section_3 = [] %}
-
-{# First we parse the other_links, we make a simple grid. #}
-{% for section in site_specific_footer.other_links %}
-  {% if loop.index is odd %}
-    {% set _section_2 = _section_2|merge([{
-      'title': section.label,
-      'links': bcl_footer_links(section.links),
-    }]) %}
-  {% else %}
-    {% set _section_3 = _section_3|merge([{
-      'title': section.label,
-      'links': bcl_footer_links(section.links),
-    }]) %}
-  {% endif %}
-{% endfor %}
-
-{# When follow section is present. #}
-{% if site_specific_footer.social_links is not empty %}
-  {# If section numbers are odd, then we flip the last row so to have the follow as first. #}
-  {% if _section_2|length > _section_3|length %}
-    {% set _last_in_section_2 = _section_2|last %}
-
-    {# Compensate for bug in twig where _section_2[1:] returns empty if _section_2|length is 1. #}
-    {% if _last_in_section_2 is empty %}
-      {% set _last_in_section_2 = _section_2[0] %}
-      {% set _section_2 = [] %}
-    {% endif %}
-
-    {% set _section_2 = _section_2[:_section_2|length - 1] %}
-    {% set _section_3 = _section_3|merge([_last_in_section_2]) %}
-
-  {% endif %}
+{% extends "@oe_whitelabel/overrides/navigation/oe-corporate-blocks-neutral-footer.html.twig" %}
+{% block content %}
+  {% set _top_sections =  {
+    'extra_classes': 'pt-4 pt-lg-5',
+    'cols': [
+      {
+        'classes': 'col-12 col-lg-4',
+        'items': [
+        {
+          'type': 'content',
+          'content': footer_top_left_content,
+        },
+      ],
+      },
+      {
+        'classes': 'col-12 col-lg-4 pb-4 pb-lg-0',
+        'items': _section_2,
+      },
+      {
+        'classes': 'col-12 col-lg-4',
+        'items': _section_3,
+      }
+    ],
+  } %}
 
-  {# Follow section must always be the last on the left. #}
-  {% set _section_2 = _section_2|merge([{
-    'title': 'Follow us on social media'|t,
-    'links': bcl_footer_links(site_specific_footer.social_links),
-  }]) %}
-{% endif %}
+  {% set eu_logo %}
+    <picture>
+      <source media="(max-width: 992px)"
+              srcset="{{ bcl_logo_path ~ 'eu/mobile/logo-eu--' ~ current_language_id|to_internal_language_id ~ '.svg' }}">
```

Comment:
some rebasing is required, this filter `to_internal_language_id` is missing in this branches code